### PR TITLE
fix: unshallow to accept repoUrl

### PR DESCRIPTION
**Pull request by botcheddevil**

### CODE REVIEW

File: lib/git.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ async function isRefInHistory(ref) {
 /**
  * Unshallow the git repository (retriving every commits and tags).
  */
-async function unshallow() {
-  await execa('git', ['fetch', '--unshallow', '--tags'], {reject: false});
+async function unshallow(origin) {
```
Comment (pvdlg) [inline lib/git.js]: Can you rename the `origin` parameter to `repositoryUrl` and add the JSDoc?

Comment (botcheddevil) [reply lib/git.js]: done

**Comment by pvdlg** [comment]
Thanks!

**Comment by pvdlg** [comment]
Fix #764

**Pull request merged**